doc/source/development/contributing.rst

```diff
@@ -1275,8 +1275,9 @@ Performance matters and it is worth considering whether your code has introduced
 performance regressions. pandas is in the process of migrating to
 `asv benchmarks <https://github.com/spacetelescope/asv>`__
 to enable easy monitoring of the performance of critical pandas operations.
-These benchmarks are all found in the ``pandas/asv_bench`` directory.  asv
-supports both python2 and python3.
+These benchmarks are all found in the ``pandas/asv_bench`` directory, and the
+test results can be found `here <https://pandas.pydata.org/speed/pandas/#/>`__.
+asv supports both python2 and python3.
```

Comment:
```suggestion
```

(just noticed it from reading the diff: that's no longer relevant, since we are only on python 3)